### DISCUSSION TITLE
middleware/what/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1847] [Announcements]
# middleware/what/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/what/

**Comment by liyuerich** [comment]
能否把mongoDB数据库的介绍链接进来？

**Reply by Michelle951** [reply]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/discussions/1847

**Comment by liyuerich** [comment]
在下方流程流程图中点击相应文字可以直接跳转到对应的操作指南页面。

除了最下面的一层是跳到对应的页面， 其他都是rabbitmq的页面？

**Reply by Michelle951** [reply]
" 其他都是rabbitmq的页面？"——这是因为，流程图是以rabbitmq为例，介绍各个中间件的学习路径

“最下面的一层是跳到对应的页面”——这是因为，最下面一层是跳转到各中间件的快速按钮，和上方的流程图是相互分离的两部分内容。

为更清晰地划分这两部分，已经通过小标题进行优化，见 pull request: https://github.com/DaoCloud/DaoCloud-docs/discussions/1847
